screens/auth/LoginScreen.js

```diff
@@ -107,23 +127,77 @@ export default class Login extends React.Component {
       .catch(err => console.error(err));
   }
 
+  onFocus(index) {
+    this.setState({
+      [this.state.booleans[index]]: true
+    });
+  }
+
+  onBlur(index) {
+    this.setState({
+      [this.state.booleans[index]]: false
+    });
+  }
+
   render() {
     return (
       <LoginContainer>
-        <Input
-          placeholder="Phone Number (i.e. 1234567890)"
-          keyboardType="number-pad"
-          maxLength={10}
-          value={this.state.phoneNumber}
-          onChangeText={text => this.setState({ phoneNumber: text })}
-        />
-        <Input
-          placeholder="Password"
-          secureTextEntry
-          onChangeText={text => this.setState({ password: text })}
-          value={this.state.password}
-        />
-        <Button title="Log In" onPress={() => this.handleSubmit()} />
+        <BigTitle>Log in</BigTitle>
+        <InputsContainer>
+          <InputContainer>
+            <Caption
+              color={
+                this.state.phoneNumBool
+                  ? Colors.primaryGreen
+                  : Colors.activeText
+              }>
+              {' '}
+              Phone Number
+            </Caption>
+            <Input
+              onBlur={() => this.onBlur(0)}
+              onFocus={() => this.onFocus(0)}
+              placeholder="513-668-6868"
+              keyboardType="number-pad"
+              maxLength={10}
+              value={this.state.phoneNumber}
+              onChangeText={text => this.setState({ phoneNumber: text })}
+              bcolor={
+                this.state.phoneNumBool
+                  ? Colors.primaryGreen
+                  : Colors.activeText
+              }
+            />
+          </InputContainer>
+
+          <InputContainer>
+            <Input
+              onBlur={() => this.onBlur(1)}
+              onFocus={() => this.onFocus(1)}
+              placeholder="Password"
+              secureTextEntry
+              onChangeText={text => this.setState({ password: text })}
+              value={this.state.password}
+              color={
+                this.state.passwordBool
+                  ? Colors.primaryGreen
+                  : Colors.activeText
+              }
+            />
+          </InputContainer>
+        </InputsContainer>
+        <JustifyCenterContainer>
+          <LoginButtonContainer>
+            <FilledButtonContainer
+              width="291px"
+              onPress={() => this.handleSubmit()}>
+              <ButtonLabel color="white">LOG IN</ButtonLabel>
+            </FilledButtonContainer>
+          </LoginButtonContainer>
+          <ButtonContainer>
+            <Body color={Colors.primaryGreen}>Forgot password?</Body>
```

Comment:
Add some top margin (maybe 12px) above this Forgot password? button

Reply:
Added bottom margin for the `LoginButtonContainer` for spacing between the login button and forgot password button. Does it still need additional margin?

Reply:
^ oops just your comment below regarding bottom margin.